library/spdm_requester_lib/libspdm_req_send_receive.c

```diff
@@ -102,6 +102,12 @@ libspdm_return_t libspdm_send_request(void *spdm_context, const uint32_t *sessio
     if (LIBSPDM_STATUS_IS_ERROR(status)) {
         LIBSPDM_DEBUG((LIBSPDM_DEBUG_INFO, "transport_encode_message status - %p\n",
                        status));
+        if ((session_id != NULL) &&
+            ((status == LIBSPDM_STATUS_SEQUENCE_NUMBER_OVERFLOW) ||
+             (status == LIBSPDM_STATUS_LOW_ENTROPY) ||
```

Comment:
`LIBSPDM_STATUS_LOW_ENTROPY` and `LIBSPDM_STATUS_CRYPTO_ERROR` aren't fatal to the session. The Requester could try again, maybe after more entropy has been collected by the system.

Reply:
I dropped LIBSPDM_STATUS_LOW_ENTROPY check.

For crypto error, we have discussed in https://github.com/DMTF/libspdm/issues/2082. We should terminate the session.